### PR TITLE
Yg update

### PR DESCRIPTION
**Pull request by yasirinitialcode**
# Description
Flutter example project has been updated.

# Output

https://github.com/Netmera/Netmera-Flutter-Example/assets/115461849/77554ca6-701f-43fa-9016-86ce93fc274c

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,7 +45,8 @@ app.*.map.json
 /android/app/profile
 /android/app/release
 
+.env
 /android/app/agconnect-services.json
 /android/app/google-services.json
 /ios/Runner/GoogleService-Info.plist
-/ios/Runner/keys.plist
+/ios/NetmeraTSExample/GoogleService-Info.plist
```
Comment (bakiinitialcode) [inline .gitignore]: Wrong project 😄

File: android/app/src/main/kotlin/com/netmera/flutterexample/MainActivity.kt [hunk 2]
```diff
@@ -4,7 +4,61 @@
...
+
+    override fun configureFlutterEngine(flutterEngine: FlutterEngine) {
+        super.configureFlutterEngine(flutterEngine)
+
+        MethodChannel(
+            flutterEngine.dartExecutor.binaryMessenger,
+            CHANNEL
+        ).setMethodCallHandler { call, result ->
+            when (call.method) {
+                "setApiKey" -> {
```
Comment (bakiinitialcode) [inline android/app/src/main/kotlin/com/netmera/flutterexample/MainActivity.kt]: Lets made method names `Enum`

File: android/app/src/main/kotlin/com/netmera/flutterexample/MainActivity.kt [hunk 3]
```diff
@@ -4,7 +4,61 @@
...
+                    val apiKey = call.argument<String>("apiKey")
```
Comment (bakiinitialcode) [inline android/app/src/main/kotlin/com/netmera/flutterexample/MainActivity.kt:33]: Lets made argument names constant

**Review by bakiinitialcode — COMMENTED** [review]
Please see my inline comments. Also can we change the button colors to blue to match our other projects?

**Review by bakiinitialcode — APPROVED** [review]

**Review by yunusinitialcode — APPROVED** [review]

**Pull request merged**